rollup.config.js

```diff
@@ -1,44 +1,54 @@
 import nodeResolve from 'rollup-plugin-node-resolve';
 import babel from 'rollup-plugin-babel';
-import replace from 'rollup-plugin-replace';
 import uglify from 'rollup-plugin-uglify';
+import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
+import pkg from './package.json';
 
-var env = process.env.NODE_ENV;
+const input = './src/index.js';
 
-var config = {
-  output: {
-    name: 'ReactInputMask',
-    globals: {
-      react: 'React'
-    }
+const isExternal = id => !id.startsWith('\0') && !id.startsWith('.') && !id.startsWith('/');
+
+export default [
+  {
+    input,
+    output: {
+      file: 'dist/react-input-mask.js',
+      format: 'umd',
+      name: 'ReactInputMask',
+      globals: { react: 'React' }
+    },
+    external: ['react'],
+    plugins: [
+      nodeResolve(),
+      babel(),
+      sizeSnapshot()
+    ]
   },
-  external: ['react'],
-  plugins: [
-    nodeResolve({
-      jsnext: true
-    }),
-    babel(),
-    replace({
-      'process.env.NODE_ENV': JSON.stringify(env)
-    })
-  ]
-};
 
-if (env === 'production') {
-  config.plugins.push(
-    uglify({
-      compress: {
-        warnings: false,
-        ie8: true
-      },
-      mangle: {
-        ie8: true
-      },
-      output: {
-        ie8: true
-      }
-    })
-  );
-}
+  {
+    input,
+    output: {
+      file: 'dist/react-input-mask.min.js',
+      format: 'umd',
+      name: 'ReactInputMask',
+      globals: { react: 'React' }
+    },
+    external: ['react'],
+    plugins: [
+      nodeResolve(),
+      babel(),
+      uglify({
+        compress: { warnings: false, ie8: true },
+        mangle: { ie8: true },
+        output: { ie8: true }
+      })
+    ]
+  },
 
-export default config;
+  {
+    input,
+    output: { file: pkg.main, format: 'cjs' },
+    external: isExternal,
```

Comment:
Why does cjs build need `isExternal` function instead of array?

Reply:
It's universal function to exclude any dependency (for example injected with babel). I hope soon it will be added in rollup itself.

Reply:
Do we need it now or it's a safety change in case we have more external dependencies in future? Because the only external we currently have is React and Babel doesn't inject dependencies unless we use  `babel-plugin-transform-runtime`.

Reply:
Yes, it's a basis for future PRs.

Reply:
I propose to not use isExternal in this PR then. More externals would lead to more peerDependencies and it would be a serious change anyway. So it's better to discuss it when we actually need it.

Reply:
Ah, now I got it, thanks. Could you put some comments about the reason into config? It looks a little "magic" now.

Reply:
Added